pkgs/development/compilers/vlang/default.nix

```diff
@@ -1,20 +1,20 @@
 { lib, stdenv, fetchFromGitHub, glfw, freetype, openssl, makeWrapper, upx, boehmgc, xorg, binaryen, darwin }:
 
 let
-  version = "weekly.2023.44";
+  version = "0.4.3";
   ptraceSubstitution = ''
     #include <sys/types.h>
     #include <sys/ptrace.h>
   '';
   # Required for bootstrap.
   vc = stdenv.mkDerivation {
     pname = "v.c";
-    version = "unstable-2023-10-30";
+    version = "0.4.3";
     src = fetchFromGitHub {
       owner = "vlang";
       repo = "vc";
-      rev = "66b89ab916c13c5781753797d1f4ff08e427bb6b";
-      hash = "sha256-5Y7/rlcoIHjbf79A1rqFysNFc5+p6CY09MRPQalo7Ak=";
+      rev = "5e691a82c01957870b451e06216a9fb3a4e83a18";
```

Comment:
This is a nitpick - Maybe add a note here that the person updating ``vlang`` should make sure to get the ``v.c`` commit that corresponds with ``vlang``'s version, if possible.

Reply:
Maybe an upstream issue should be made requesting that ``v.c`` is tagged and releases are made to correspond with ``vlang``.

Reply:
> Maybe an upstream issue should be made requesting that ``v.c`` is tagged and releases are made to correspond with ``vlang``.

That is definitely a better idea.

Reply:
> This is a nitpick - Maybe add a note here that the person updating `vlang` should make sure to get the `v.c` commit that corresponds with `vlang`'s version, if possible.

I've added the following comments:

    # vc is the V compiler's source translated to C (needed for boostrap).
    # So we fix the rev corresponding to the V version above.